theme_nano/views/snippets/s_parallax.xml

```diff
@@ -4,20 +4,11 @@
 <template id="s_parallax" inherit_id="website.s_parallax">
     <!-- Section -->
     <xpath expr="//section" position="attributes">
-        <attribute name="class" add="pt48 pb48" separator=" "/>
-        <attribute name="data-oe-shape-data">{"shape":"web_editor/Rainy/10", "flip":["x","y"]}</attribute>
+        <attribute name="class" add="pt48 pb48 o_not_editable" separator=" "/>
```

Comment:
Why do we need `.o_not_editable` here? That looks like something we'd want on the base snippet if it's needed?

Reply:
I think you're right and we should remove this class. By default the `s_parallax` is meant to receive other snippets within its DOM but in this specific scenario I wanted to use it as a "banner" without being editable. But the more I think about it the more I think you're right, it doesn't make sense to force the class here if it's not the case on the default snippet, and restricting the edition makes no sense as it's something the user might want to do.